### DISCUSSION TITLE
Will the features from Otaku Testing eventually come to regular Otaku?

### DISCUSSION THREAD

**Discussion by Rinicko100** [Goldenfreddy0703/Otaku-Testing#11] [Q&A] [answered]
Hello, 

I'm just wondering if the features from this addon will eventually be added to the og Otaku, or is this addon going to be the main one?

I really like some features that have been added here, like the dub dates, dub labels, MAL database, etc. But I'm trying to avoid having to change every single widget and shortcut on my setup since I have a lot, so I'm just wondering.

Edit
Apologies for adding this under issues,  I couldn't find a discussion tab

**Comment by Goldenfreddy0703** [comment, marked as answer]
I doubt it cause main reason why is cause of making the addon compatible with python 2 and it would be a mess and then there is reuse language invoker and all, its just not worth it. 

This addon is probably going to be the main one eventually but not for a long time.

Also i forgot to add the discussion tab, thanks for pointing that out.

**Reply by Sickhead01** [reply]
i really like the "Choose Sorting.." setting. It would be a great addition to the main addon imo. I like how i can make it lists all the dubs regardless of the source first, including the embeds. If you don't want to add it can you all implement the RD fix make Otaku Testing not HAVE to Scrape Uncached Sources like you just did for the main one? Because for some older shows not enough seeders are on the torrents for the caching process to even happen or they're stuck at zero or extremely show. If Otaku Testing had that i'd just stick with it permanently
